### DISCUSSION TITLE
Is there a way to forcefully delete a corrupted podcast?

### DISCUSSION THREAD

**Discussion by YamiYukiSenpai** [thrillfall/nextcloud-gpodder#158] [Q&A]
For some strange reason, some of my podcast got corrupted.  For most of them, I managed to delete them by logging out and deleting the ones that aren't syncing, except for Podcast Unlocked:
![image](https://github.com/thrillfall/nextcloud-gpodder/assets/13153124/404d0db6-14e5-47c9-84ce-7d972f5d62be)

Any ideas on how to properly fix this?

**Comment by thrillfall** [comment]
What do you mean by corrupted?

**Reply by YamiYukiSenpai** [reply]
the Podcast Unlocked doesn't refresh.  Every time I sync, a new Podcast Unlocked subscription gets added instead.

**Reply by YamiYukiSenpai** [reply]
![Screenshot_20240222_065032_AntennaPod](https://github.com/thrillfall/nextcloud-gpodder/assets/13153124/87e24ad3-d931-4ee1-8cc2-8ad29cc53810)

**Reply by YamiYukiSenpai** [reply]
Strange that from KDE's Kasts, it's fine:
![image](https://github.com/thrillfall/nextcloud-gpodder/assets/13153124/1edf9e0a-af06-4760-9929-14f5be9645fe)

**Comment by JonOfUs** [comment]
Sounds like the feed URL of the podcast changed - you could try to manually change the feed URL to the new one directly in AntennaPod.
On the Podcast page, click on the "Info" icon and then on the three dots in the top right to change the feed URL.

**Reply by YamiYukiSenpai** [reply]
Confirmed to be identical

![Screenshot_20240222_192829_AntennaPod](https://github.com/thrillfall/nextcloud-gpodder/assets/13153124/b6fc03dd-410e-49dc-a33f-1ee337278437)

https://feeds.megaphone.fm/ignunlocked

https://corp.ign.com/feeds

**Reply by JonOfUs** [reply]
Do I understand it correctly that both versions of the "Podcast Unlocked" podcast inside AntennaPod have the same feed URL, "https://feeds.megaphone.fm/ignunlocked"? That is very weird...

**Reply by YamiYukiSenpai** [reply]
yea..tell me about it. I tried to delete all entries, but it kept refreshing.  Play time doesn't even update, and it just creates a new one, and without the playtime.

**Reply by YamiYukiSenpai** [reply]
Strange
I tested from KDE's Kasts, and it's fine:
![image](https://github.com/thrillfall/nextcloud-gpodder/assets/13153124/4aed8870-aefb-425a-9b1d-46a38e22dae1)
